src/page-modules/assistant/server/journey-planner/journey-gql/trip-with-details.gql

```diff
@@ -82,6 +96,9 @@ query TripsWithDetails(
         intermediateEstimatedCalls {
           aimedDepartureTime
         }
+        situations {
+          ...situation
```

Comment:
Is everything in `situation` needed?

Reply:
We can remove ID, advice, and infoLinks. Should update that for all situations in the app then

Reply:
Instead of removing those, I added them to the situation dialog instead to have the same information as in the app.

Reply:
Nice, that's fine 👏